modules/caddyhttp/opentelemetry/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry module
+
+This module provides integration with OpenTelemetry tracing facilities. It is implemented
+as `caddyhttp.MiddlewareHandler` and can be chained into a list of other handlers.
+
+When enabled, it propagates an existing tracing context or will init a new one otherwise.
+
+It is based on `https://github.com/open-telemetry/opentelemetry-go`.
+
+## Configuration
+
+### Environment variables
+
+It can be configured using environment variables defined
+by [spec](https://github.com/open-telemetry/opentelemetry-specification).
+
+IMPORTANT: Please, consider the version
+of [https://github.com/open-telemetry/opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go). Some parts
+of the specification may be not implemented yet.
```

Comment:
The metrics and logs parts of the OTel spec are non-stable, but Tracing is stable so I'm not sure if this statement is accurate. Either way, it adds a bit of confusion that I'm not sure is necessary in this context.

Reply:
The reason why I put here this statement is this ticket: https://github.com/open-telemetry/opentelemetry-go/issues/1698. 

I just wanted to emphasize that customers should not expect that, for example, variable OTEL_TRACES_SAMPLER will change the behavior of the tracing.

What do you think, does it make sense to add this clarification or it is better to remove this line?